torchvision/transforms/v2/_geometry.py

```diff
@@ -164,25 +305,23 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
 
 
 class FiveCrop(Transform):
-    """
-    Example:
-        >>> class BatchMultiCrop(transforms.Transform):
```

Comment:
The fact that `BatchMultiCrop`  was still there makes me think this example is largely outdated. I removed it, we can add it back if there's an up-to-date version somewhere.

Reply:
Nope. I just updated it in the migration PR after we got rid of the labels datapoint. What makes you think it is outdated?

Reply:
I went (way) too fast, sorry. I'll put it back.